src/lib/core/ripple/ripple.ts

```diff
@@ -7,21 +7,22 @@ import {
   OnChanges,
   SimpleChanges,
   OnDestroy,
-  OpaqueToken,
+  InjectionToken,
   Optional,
 } from '@angular/core';
 import {RippleConfig, RippleRenderer} from './ripple-renderer';
 import {ViewportRuler} from '../overlay/position/viewport-ruler';
 import {RippleRef} from './ripple-ref';
 
-/** OpaqueToken that can be used to specify the global ripple options. */
-export const MD_RIPPLE_GLOBAL_OPTIONS = new OpaqueToken('md-ripple-global-options');
-
 export type RippleGlobalOptions = {
   disabled?: boolean;
   baseSpeedFactor?: number;
 };
 
+/** Injection token that can be used to specify the global ripple options. */
+export const MD_RIPPLE_GLOBAL_OPTIONS = new InjectionToken<RippleGlobalOptions>(
```

Comment:
Generally prefer line breaking at the higher syntactic level, e.g.:
```ts
export const MD_RIPPLE_GLOBAL_OPTIONS = 
    new InjectionToken<RippleGlobalOptions>('md-ripple-global-options');
```
(at the assignment)